discord-bot/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I didn't even know it was legal to have a package.json with no package name.

Reply:
Please don't tell the cops.